SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/ObjectSpy.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include <Utils/RPCContainer.h>
+
+#include <CoreMinimal.h>
+
+#include "ObjectSpy.generated.h"
+
+TArray<uint8> TypeToArray(ESchemaComponentType Type);
```

Comment:
These functions should live in a class or namespace.

Reply:
Yes, it will be cleaner to put those in a namespace.